doc/source/visualization.rst

```diff
@@ -622,6 +645,10 @@ See the :meth:`scatter <matplotlib.axes.Axes.scatter>` method and the
 Hexagonal Bin Plot
 ~~~~~~~~~~~~~~~~~~
 
+.. versionadded:: 0.17.0
+
+You can create hexagonal bin plots with :meth:`DataFrame.plot.hexbin`.
+
 .. versionadded:: 0.14
 
 You can create hexagonal bin plots with :meth:`DataFrame.plot` and
```

Comment:
Ah other plots section also have duplicated descriptions. Maybe for ``versionadded``? 

Personally, I think ``versionadded`` for ``.plot.<>`` accessor are not needed in each plot kind, because this is described in the top of this document. @jorisvandenbossche ?